framework/web/UrlRule.php

```diff
@@ -217,6 +217,21 @@ public function parseRequest($manager, $request)
 
         $pathInfo = $request->getPathInfo();
         $suffix = (string) ($this->suffix === null ? $manager->suffix : $this->suffix);
+
+        if (!empty($pathInfo)) {
+            if (substr_compare($pathInfo, '/', -1, 1) === 0) {
+                if ($suffix === '') {
```

Comment:
Does it make sense to have separate URL rules for `http://exampe.com/contact` and `http://exampe.com/contact/`?